react-design/my-app/src/RotatingBanner.jsx

```diff
@@ -1,19 +1,16 @@
 import Banner from './Banner';
 import NextBtn from './NextBtn';
 import PrevBtn from './PrevBtn';
-import Carousel from './Carousel';
+import Indicators from './Indicators';
 
-export default function RotatingBanner() {
+export default function RotatingBanner({ items }) {
+  // const [count, setCount] = useState(0);
```

Comment:
Uncomment, in Phase 2, we want to set up the state and test if it works!

Reply:
A better name would be because it is more clear: 
` const [currentIndex, setCurrentIndex] = useState(0);`
 
We want to know what the current index is so we display the correct number we currently are on(aka know where in the items array we currently displaying.